packages/repo-tools/package.json

```diff
@@ -33,17 +33,22 @@
     "@backstage/cli-common": "workspace:^",
     "@backstage/errors": "workspace:^",
     "@manypkg/get-packages": "^1.1.3",
-    "@microsoft/api-documenter": "^7.17.11",
-    "@microsoft/api-extractor": "^7.23.0",
-    "@microsoft/api-extractor-model": "^7.17.2",
-    "@microsoft/tsdoc": "0.14.1",
+    "@microsoft/api-documenter": "^7.19.26",
+    "@microsoft/api-extractor": "^7.33.7",
+    "@microsoft/api-extractor-model": "^7.25.3",
+    "@microsoft/tsdoc": "^0.14.2",
+    "@microsoft/tsdoc-config": "~0.16.1",
+    "@rushstack/node-core-library": "^3.53.3",
+    "@spotify/prettier-config": "^14.1.3",
     "chalk": "^4.0.0",
     "commander": "^9.1.0",
     "fs-extra": "10.1.0",
     "glob": "^8.0.3",
     "is-glob": "^4.0.3",
     "minimatch": "^5.1.1",
-    "ts-node": "^10.0.0"
+    "prettier": "^2.8.1",
```

Comment:
Perhaps make this a peer dep as well and only run it when available? Iirc the main purpose of running prettier is to have the API reports properly formatted out of the box to not have to run prettier manually afterwards. If prettier isn't installed that's not really a concern. It also makes it important to run it with the correct config.